justfile

```
@@ -4,10 +4,16 @@ help:
 
 # Prepare the environment for development, installing all the dependencies and
 # setting up the pre-commit hooks.
-setup:
+setup: setup-tket-c-api
     uv sync
     [[ -n "${TKET2_JUST_INHIBIT_GIT_HOOKS:-}" ]] || uv run pre-commit install -t pre-commit
 
+# Configure the conan cache for tket-c-api.
+setup-tket-c-api:
+    #!/usr/bin/env bash
+    uv run conan remote add tket-libs https://quantinuumsw.jfrog.io/artifactory/api/conan/tket1-libs --index 0
```

```suggestion
    uv run conan remote add -f tket-libs https://quantinuumsw.jfrog.io/artifactory/api/conan/tket1-libs --index 0
```

By adding `-f` this should not error if already present.